public/app/plugins/panel/alertlist/UnifiedAlertList.tsx

```diff
@@ -407,12 +407,21 @@ export const getStyles = (theme: GrafanaTheme2) => ({
     white-space: nowrap;
     overflow: hidden;
     text-overflow: ellipsis;
+    container: instance / inline-size;
   `,
   customGroupDetails: css`
     margin-bottom: ${theme.spacing(0.5)};
   `,
   link: css`
     word-break: break-all;
     color: ${theme.colors.primary.text};
+    display: flex;
+    align-items: center;
+    gap: ${theme.spacing(1)};
+  `,
+  viewRuleLinkText: css`
+    @container (max-width: ${theme.spacing(40)}) {
+      display: none;
+    }
```

Comment:
For some reason some tests throw an error and it seems it's due to the `@container` rule. Can we use `@media` instead? This doesn't output any error. 
```suggestion
viewRuleLinkText: css({
    [`@media (max-width: ${theme.spacing(40)})`]: {
      display: 'none',
    },
  }),
```

Reply:
It looks like our tooling doesn't support container queries yet. Unfortunately, `@media` doesn't solve the problem - The issue occurs when your panel width is small, which can happen not only when you have a small viewport but also when you have other panels next to the Alert List one.
I've changed the code to use the `width` property every panel receives in props to apply styles conditionally based on the panel size